### PR TITLE
Make designers happy again

### PR DESCRIPTION
**Pull request by flexsurfer**

### CODE REVIEW

**Review by janherich — APPROVED** [review]

**Review by dmitryn — APPROVED** [review]

**Comment by asemiankevich** [comment]
@flexsurfer any hint how this PR is making @denis-sharypin happy and which screen? :) appreciate that!

**Comment by denis-sharypin** [comment]
@asemiankevich just approve it :)

**Comment by asemiankevich** [comment]
@denis-sharypin anything that makes u happy is my prio ) Approved)

File: src/status_im/ui/screens/profile/components/views.cljs [hunk 1]
```diff
@@ -32,19 +32,21 @@
...
-                                      :edit?   false}]
-   [react/view styles/profile-header-name-container
-    [react/text {:style           styles/profile-name-text
-                 :font            :medium
-                 :number-of-lines 1}
-     name]
-    (when (:public-key contact)
-      [react/text {:style styles/profile-three-words
-                   :font :medium
+  (let [pk (:public-key contact)
```
Comment (yenda) [inline src/status_im/ui/screens/profile/components/views.cljs]: use public-key no shorcut please

**Review by yenda — APPROVED** [review]
can you add a short description to the PR ?

**Pull request merged**